src/index.ts

```diff
@@ -397,6 +397,19 @@ const cs3browser: JupyterFrontEndPlugin<void> = {
         fileBrowser.useFuzzyFilter = useFuzzyFilter;
       });
 
+    /**
+     * cs3api4lab modification - loads plugin settings to redirect user to user directory at start
+     */
+    fileBrowser.model.restored.then(() => {
```

Comment:
I would even add a "home" button in the toolbar for users to get to the starting page more easily.

Reply:
You are correct, first / is requested and then it is switched to user dir. 
I would add another button in separate task though.

Reply:
Yes, done #168 

For the cd, can't we change the first directory instead of doing cd? would that mean big changes or not really?

Reply:
@diocas A button in the toolbar for redirecting to user home dir is even better solution than automatic redirection. Especially when file browser automatically changes the directory to an opened document and remembers the path of the directory when the page is refreshed. So if you button instead of automatic redir it's fine with me.

Reply:
The button is complementary, we should always start in home if there is no other path loaded. But this is a good point, with this cd we're not even taking that in consideration, right? Can't you see how they do it and do the same?
I would expect the ui to open in some path if it's given, or in home (or `/`) is not.

Reply:
@diocas I tested this on the dev environment and it is taking other redirection into consideration when the document is opened or the user changed directories and refreshed the page our redirection will be skipped and it will switch to the current directory. Only thing that I'm worried about is loading times, I can take a look if there is a way to control this better but the default behavior is coded into file browser plugin and I'm not sure we can fully control it. I will take a look what else we can do with that.